airbyte-integrations/connectors/source-postgres/build.gradle

```diff
@@ -0,0 +1,33 @@
+plugins {
+    id 'application'
+}
+
+apply from: rootProject.file('tools/gradle/commons/integrations/image.gradle')
+apply from: rootProject.file('tools/gradle/commons/integrations/integration-test.gradle')
+
+dependencies {
+    implementation "org.postgresql:postgresql:42.2.18"
+
+    implementation project(':airbyte-db')
+    implementation project(':airbyte-integrations:bases:base-java')
+    implementation project(':airbyte-protocol:models')
+    implementation project(':airbyte-integrations:connectors:source-jdbc')
+
+    testImplementation 'org.apache.commons:commons-text:1.9'
+    testImplementation 'org.apache.commons:commons-lang3:3.11'
+    testImplementation 'org.apache.commons:commons-dbcp2:2.7.0'
+    testImplementation 'com.fasterxml.jackson.core:jackson-databind'
```

Comment:
why is this needed? isn't it imported to all of our java gradle projects automatically?